src/main/java/org/gridsuite/modification/server/modifications/EquipmentDeletion.java

```diff
@@ -36,7 +44,17 @@ public void apply(Network network, Reporter subReporter) {
         if (identifiable instanceof Connectable) {
             new RemoveFeederBay(modificationInfos.getEquipmentId()).apply(network, true, subReporter);
         } else if (identifiable instanceof HvdcLine) {
-            ((HvdcLine) identifiable).remove();
+            List<String> shuntCompensatorIds = Stream.concat(
+                    modificationInfos.getMcsOnSide1() != null ? modificationInfos.getMcsOnSide1().stream() : Stream.of(),
```

Comment:
I don't undestand why we tranfert from front only selected ?

Reply:
you mean  "why we dont transfer only select" ?

I wanted to store them, because when a user edit its modification, he will get exactly what it saw. In edition, I dont want the form to be built dynamically.